en/docs/setup/migrating-to-590.md

```diff
@@ -90,9 +83,13 @@ Follow the steps below to perform the upgrade.
     `          <NEW_IS_HOME>/repository/components/dropins         `
     directory.
     
-    !!! note
+    !!! warning "Important" 
         You may need to update the custom components to work with WSO2 Identity Server 5.9.0, 
         refer [Migrating custom components](../../setup/migrating-preparing-for-migration/#migrating-custom-components).
+        If applicable migrate
```

Comment:
If applicable **,** migrate [Data Publishers](../../setup/migrating-data-publishers) and [Custom Scope validators](../../setup/migrating-custom-scope-validators) **by** referring to the relevant document.

Reply:
fixed